control/controldisplay/formatter.go

```diff
@@ -63,37 +60,23 @@ type Formatter interface {
 	FileExtension() string
 }
 
-func GetExportFormatter(exportFormat string) (Formatter, error) {
-	formatter, found := exportFormatters[exportFormat]
-	if !found {
-		return nil, fmt.Errorf("invalid export format '%s' - must be one of %s", exportFormat, exportFormatters.keys())
-	}
-	return formatter, nil
+func GetDefinedExportFormatter(arg string) (Formatter, bool) {
+	formatter, found := exportFormatters[arg]
+	return formatter, found
 }
 
-func GetOutputFormatter(outputFormat string) (Formatter, error) {
-	formatter, found := outputFormatters[outputFormat]
-	if !found {
-		return nil, fmt.Errorf("invalid output format '%s' - must be one of %s", outputFormat, outputFormatters.keys())
+func GetTemplateExportFormatter(arg string, allowFilenameEvaluation bool) (Formatter, string, error) {
+	templateFormat, fileName, err := GetExportTemplate(arg, allowFilenameEvaluation)
+	if err != nil {
+		return nil, "", err
 	}
-	return formatter, nil
+	formatter, err := CreateTemplateFormatter(*templateFormat)
```

Comment:
CreateTemplateFormatter -> NewTemplateFormatter, move to just after struct defintion

Reply:
also, move `GetExportTemplate` to `export_template.go`

also I find the naming/code structure as bit confusing - remind me what the returned filename is again?